src/transformers/modeling_flax_utils.py

```diff
@@ -1029,6 +1104,7 @@ def save_pretrained(
         push_to_hub=False,
         max_shard_size="10GB",
         token: Optional[Union[str, bool]] = None,
+        safe_serialization: bool = False,
```

Comment:
This is kept to `False` by default for now; switching it to `True` is possible in the short term, but we would need to support sharded weights first.

Reply:
See https://github.com/huggingface/transformers/blob/84724efd101af52ed3d6af878e41ff8fd651a9cc/src/transformers/modeling_flax_utils.py#L93